snapcraft.yaml

```diff
@@ -1,30 +1,42 @@
 name: ktor
-summary: Creates porjects with ktor framework setup
-version: '2.0.0-eap-5'
-base: core18
-grade: stable
-confinement: classic
+base: core24
+version: 'dev'
+summary: Generates Ktor applications through the command line interface.
+website: https://github.com/ktorio/ktor-cli
+license: Apache-2.0
 description: |
-  Ktor is a web application framework for creating connected systems.
-  You can use it to create server-side as well as client-side applications.
-  It supports multiple platforms, including JVM, JavaScript, and Kotlin/Native.
-  This CLI tool provides functionality to create new Ktor projects.
+  Ktor is an asynchronous framework for creating microservices, web applications and more.
+  Written in Kotlin from the ground up.
+
```

Comment:
Maybe we need to add here that this is the CLI tool for generating projects and specify what type of projects can be generated (client, server etc).

Reply:
We have this in summary:
> Generates Ktor applications through the command line interface.

Should we add this to the description too?

Reply:
In my understanding a summary is a summary of the description so it should be there, but I don't know how much details are necessary for this file specifically. What will the description be used for?